### ISSUE TITLE
Fmod Studio Runs Super Slowly When Loaded

### ISSUE THREAD

**Issue by Musket519** [GaijinEntertainment/fmod_studio_warthunder_for_modders#106]
This might be an issue for the FMOD forums but for some reason when I load this project into FMOD it begins to run extremely slowly. Anyone know whats up with that?

**Comment by OXEPU3M** [comment]
This one is starting extremely fast.
If you'd got a last patch project you'd know  what is really slow xD

**Comment by Uchuujin01** [comment]
What do you mean by saying "slow"? The project freezes while playing audio? Or it has huge loading time?

**Comment by Musket519** [comment]
> 

When I click on literally anything in FMod it takes about 45 seconds to actually do what I clicked on. Like it’s running at 1FPS

**Comment by ThM112** [comment]
It may be because you are using the most recent version of FMOD, try using FMOD v2.01.6

**Comment by Uchuujin01** [comment]
> > 
> 
> When I click on literally anything in FMod it takes about 45 seconds to actually do what I clicked on. Like it’s running at 1FPS

Hello! I can't say anything about this issue, never faced something like this. Maybe this is your HDD issue?

**Comment by zarusoba10** [comment]
This also happens on my PC
Both FMOD and assets are in SSD, but it runs at 1FPS.
It may be due to high CPU usage.

Note: It worked fine a year ago.

**Comment by ThM112** [comment]
No, I can promise you it is the version of FMOD you are using, I used the most recent version of FMOD and the files took forever to load and the application barely responded, I then tried to download the version of FMOD that these files are supported on and it ran like a charm, try using FMOD 2.1.6

**Comment by zarusoba10** [comment]
It was solved by changing the version. thank you.

However, since it is written as ver2.01.06 on the FMOD site, it may be better to change the readme.

**Comment by ThM112** [comment]
the 0's in the version name are negligible as they are in front of the numbers and they don't really mean anything

**Comment by zarusoba10** [comment]
It certainly doesn't make sense, but it's true that I'm confused ...
